src/en/space-station-14/art.md

```diff
@@ -1,5 +1,233 @@
+Use multiple layers to be able to compare and reverse changes easily.
+```
+
+* Try not to go overboard with too many different hues.
+Sticking with a few primary tones (_department colors are a good use for this!_) and accenting them with neutral tones helps create a cohesive asset that doesn't look busy.
+
+
+* Don't use **darken**, **burn**, or **gradient** tools for half-tones when shading.
+Darker shades of color should be intentionally chosen from your palette.
+Using gradients of color can make sprites look muddy and make them difficult to edit in the future.
+
+
+* Don't overuse **hue-shifting** when creating shades for your palette.
+Using it sparingly can lend greater intensity to your darker tones, but overuse can create an overly cartoonish appearance that looks overly fantastical.
+The color of an object should still read clearly when shifting is applied: 
+* * An apple should be red, not a deep magenta
+* * A blue shirt shouldn't be a deep purple in its shadows
+* * Regular industrial steel shouldn't be tinted cyan and blue
+
+### Lighting
+* Sprites are usually lit from the **top**. 
+For standalone object sprites, the light source may be slightly angled to either side, but generally a centered light from above is how most sprites are lit.
+
+
+* Avoid **high-contrast** lighting. 
+Although the primary light source is coming from the top, make sure the ambient light is accounted for.
+Objects should be lit in the context of being in a _well-lit room_.
+Shadows shouldn't be pitch black and highlights shouldn't be bone white.
+
+
+* Be mindful of the **luster** of a surface when adding lighting to your sprite.
+How shiny is your material?
+Cloth surfaces are generally quite matte whereas metal usually has long reflections along its length.
+Make sure the amount and contrast of the lighting being applied is appropriate for the surface being depicted.
+* * Applying a uniformly glossy lighting across lots of sprites can lead to a 'buttery' appearance that harms readability.
+* * Less is more!
+A simple sprite with minimal lighting is preferable to an over-worked sprite that reads poorly and sticks out.
+
+
+* Keep lights **temperature neutral**.
+This relates a lot to the earlier point on hue-shifting.
+Baking shadows with purple tints gives the impression of _blue ambient light_, when it's largely the opposite!
+Keeping the temperature largely neutral ensures that when colored lighting does appear in game, it doesn't clash and creates weird tones.
+
+### Outlines
+* All sprites should have **colored outlines** around them.
+This helps improve readability in the world and clarify the forms.
+Make sure that the outline is colored according to the section it surrounds, instead of being a single flat color.
+* * Outlining an entire sprite in black often looks plain and amateurish.
+Using multiple colors in your outline gives a much better result.
+
+
+* Make sure your outlines have shading as well.
```

Comment:
I am pretty sure this goes directly against how most classic sprites are shaded. It may just be best to just remove this point entirely.